automate/utils.py

```diff
@@ -1,22 +1,42 @@
 import requests
 import json
 
+# from cryptography.fernet import Fernet
+
+
+class Crypt:
+    """This is a class that handles the encryption and decryption of access token"""
+
+    def __init__(self, value):
+        self.value = value
+
+    def encrypt(self):
+        """This encrypts value using the provided key"""
+        # Todo: (mark) write encryption function with cryptography
+        pass
+
+    def decrypt(self):
+        """This decrypts value using the provided key"""
+        # Todo: (mark) write decryption function with cryptography
+        pass
+
```

Comment:
This can be removed, since I saw a class that serves the same purpose above.

Reply:
done